src/components/EntityInfo.vue

```diff
@@ -286,10 +285,9 @@ export default class EntityInfo extends Mixins(AllowableActionsMixin, CommonMixi
     window.location.assign(this.businessProfileUrl) // assume URL is always reachable
   }
 
-  /** Downloads business summary PDF. */
-  private downloadBusinessSummary (): void {
-    // FUTURE
-  }
+  // Pass prompt downloads business summary event to parent. */
+  @Emit('downloadBusinessSummary')
+  private downloadBusinessSummary (): void {}
```

Comment:
Just thinking ahead here, eventually our EntitiesInfo will be shared, so the less functionality it does the better. 
Hence, why we are having the parent (App in this case) tackle the request.

Reply:
If we share the EntityInfo component then we'll probably need to add support for a template slot for the buttons anyway (since not all apps will need the same buttons).

I'm OK with this approach for now. The risk is making App.vue too large. I think we're there now but I don't have a slick way to rearchitect it.

Reply:
Ah yea, i like that idea a lot.